### PR TITLE
Pass the spec linter to some packages

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Pass the spec linter to some packages, as requested by autobuild.

We are only changing packages that are to be submitted for SUSE Manager 4.2 GMC. The rest will come later.

For the packages used by client tools, I am defining `%license` if it doesn't exist, as we need to stay compatible with old OS.

This is WIP and merge candidate because:
- Needs to be merged before we submit GMC
- But we shouldn't merge before we check what packages we'll be submitting for GMC.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: SPEC change

- [x] **DONE**

## Test coverage
- No tests: The packages are able to build locally

- [x] **DONE**

## Links

Tracks: https://github.com/SUSE/spacewalk/issues/14840 and https://github.com/SUSE/spacewalk/issues/14841

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/spacewalk-java.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #
 # spec file for package spacewalk-java
 #
-# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
```
Comment (juliogonzalez) [inline java/spacewalk-java.spec:4]: Please note that I excluded changing the order of the dependencies, as the changes were huge, we have a lot of `%if` in this package, and I do not want to risk breaking GMC or Uyuni 2021.06.

**Review by jordimassaguerpla — APPROVED** [review]

**Comment by juliogonzalez** [comment]
@jordimassaguerpla this is not ready for review yet, as I am quite sure I will need to add more packages :-)

But now you are a "victim" for when the PR is ready :-D

**Comment by jordimassaguerpla** [comment]
yey! Packages build correctly in the acceptance tests :woman_dancing: :man_dancing:  Well done!

**Comment by juliogonzalez** [comment]
Tests passed again, so for now, it seems we are safe.

**Pull request merged**